RunFullAnalysis.sh

```diff
@@ -228,10 +228,23 @@ if $DoHFPtSpec; then
     Channel="kDplusKpipi"
   fi
   
+  cc=""
+  year=""
+  sigma=1.
+  if [ ${Cent} == "k010" -o "${Cent}" == "k3050" -o "${Cent}" == "k3050" ]; then
```

Comment:
Here `"k3050"` is present two times

Reply:
Thanks! Indeed this was a typo, fixed in f4df70a